### ISSUE TITLE
amalgam error

### ISSUE THREAD

**Issue by CJ-Wright** [xonsh/xonsh#1323]
I got this error, any ideas what it is?
```python
Traceback (most recent call last):
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 11997, in load_static_config
    conf = json.load(f)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/json/__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/json/decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 7 column 1 (char 53)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/christopher/mc/envs/xonsh/bin/xonsh", line 6, in <module>
    sys.exit(xonsh.main.main())
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 16394, in main
    args = premain(argv)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 16381, in premain
    shell = builtins.__xonsh_shell__ = Shell(**shell_kwargs)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 16125, in __init__
    kwargs.get('cacheall', False))
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 16167, in _init_environ
    debug_level=debug)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 15811, in __init__
    load_builtins(execer=self, config=config, login=login, ctx=xonsh_ctx)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 15651, in load_builtins
    builtins.__xonsh_env__ = ENV = Env(default_env(config=config, login=login))
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 12084, in default_env
    conf = load_static_config(ctx, config=config)
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 12003, in load_static_config
    print_exception()
  File "/home/christopher/mc/envs/xonsh/lib/python3.5/site-packages/xonsh/__amalgam__.py", line 5559, in print_exception
    manually_set_trace = env.is_manually_set('XONSH_SHOW_TRACEBACK')
AttributeError: '_Environ' object has no attribute 'is_manually_set'
```
for reference my conda env is
```
# packages in environment at /home/christopher/mc/envs/xonsh:
#
cycler                    0.10.0                   py35_0    defaults
fontconfig                2.11.1                        5    defaults
freetype                  2.5.5                         0    defaults
libgfortran               3.0                           0    defaults
libpng                    1.6.17                        0    defaults
libxml2                   2.9.2                         0    defaults
matplotlib                1.5.1               np111py35_0    defaults
mkl                       11.3.1                        0    defaults
ncurses                   5.9                           0    defaults
numpy                     1.11.0                   py35_0    defaults
openssl                   1.0.2g                        0    defaults
pip                       8.1.1                    py35_1    defaults
ply                       3.8                      py35_0    defaults
prompt-toolkit            1.0.0                     <pip>
prompt_toolkit            1.0.0                    py35_0    xonsh
pygments                  2.1.3                    py35_0    defaults
pyparsing                 2.0.3                    py35_0    defaults
pyqt                      4.11.4                   py35_1    defaults
python                    3.5.1                         0    defaults
python-dateutil           2.5.2                    py35_0    defaults
pytz                      2016.3                   py35_0    defaults
qt                        4.8.7                         1    defaults
readline                  6.2                          13    tacaswell
scipy                     0.17.0              np111py35_2    defaults
setproctitle              1.1.9                    py35_0    xonsh
setuptools                20.6.7                   py35_0    defaults
sip                       4.16.9                   py35_0    defaults
six                       1.10.0                   py35_0    defaults
sqlite                    3.9.2                         0    defaults
tinydb                    3.2.0                     <pip>
tk                        8.5.18                        0    defaults
wcwidth                   0.1.5                    py35_5    xonsh
wheel                     0.29.0                   py35_0    defaults
xonsh                     0.3.4                  py35_597    xonsh
xz                        5.0.5                         1    defaults
zlib                      1.2.8                         0    defaults
```

**Comment by CJ-Wright** [comment]
This might be part of #1238 Although this is not occasional, it happens with each xonsh startup.

**Comment by gforsyth** [comment]
Thanks for reporting, @CJ-Wright !  `amalgam` makes it hard to follow the traceback.  Could you set `$XONSH_DEBUG=True` and then run `xonsh` again and paste the traceback there?  It should be more helpful in diagnosing what's going on.  Thanks!

**Comment by scopatz** [comment]
This seems like you have malformed config.json file.

**Comment by scopatz** [comment]
But the error message should be nicer :)

**Comment by scopatz** [comment]
Nevermind, this is an exception during exception

**Comment by scopatz** [comment]
This is an error caused by 2541702672813b0a0fbcd03055eafab7437026da not adhering to the os.environ API

**Comment by scopatz** [comment]
Possible fix in #1327

**Comment by scopatz** [comment]
@CJ-Wright - would you mind trying this out?

**Comment by CJ-Wright** [comment]
Sure, I'll try it out before Monday.

**Comment by scopatz** [comment]
@CJ-Wright, has this been fixed?

**Comment by CJ-Wright** [comment]
@scopatz yep thank you very much!
